app/assets/javascripts/discourse/app/lib/keyboard-shortcuts.js

```diff
@@ -130,14 +133,13 @@ export default {
     }
 
     this.keyTrapper = new keyTrapper();
-    this.container = container;
```

Comment:
This assumes this "class" isn't "subclassed" and/or `this.container` isn't meant to be public